modules/optableBidAdapter.js

```diff
@@ -17,17 +17,42 @@ const BIDDER_CODE = 'optable';
 const DEFAULT_REGION = 'ca'
 const DEFAULT_ORIGIN = 'https://ads.optable.co'
 
+function getOrigin() {
+  return config.getConfig('optable.origin') ?? DEFAULT_ORIGIN;
+}
+
+function getBaseUrl() {
+  const region = config.getConfig('optable.region') ?? DEFAULT_REGION;
+  return `${getOrigin()}/${region}`
+}
+
 export const spec = {
   code: BIDDER_CODE,
   isBidRequestValid: function(bid) { return !!bid.params?.site },
   buildRequests: function(bidRequests, bidderRequest) {
-    const region = config.getConfig('optable.region') ?? DEFAULT_REGION
-    const origin = config.getConfig('optable.origin') ?? DEFAULT_ORIGIN
-    const requestURL = `${origin}/${region}/ortb2/v1/ssp/bid`
+    const requestURL = `${getBaseUrl()}/ortb2/v1/ssp/bid`
     const data = converter.toORTB({ bidRequests, bidderRequest, context: { mediaType: BANNER } });
-
     return { method: 'POST', url: requestURL, data }
   },
+  buildPAAPIConfigs: function(bidRequests) {
+    const origin = getOrigin();
+    return bidRequests
+      .filter(req => req.ortb2Imp?.ext?.ae)
+      .map(bid => ({
+        bidId: bid.bidId,
+        config: {
+          seller: origin,
+          decisionLogicURL: `${getBaseUrl()}/paapi/v1/ssp/decision-logic.js?origin=${bid.params.site}`,
+          interestGroupBuyers: [origin],
+          perBuyerMultiBidLimits: {
+            [origin]: 100
+          },
+          perBuyerCurrencies: {
+            [origin]: 'USD'
+          }
+        }
+      }))
+  },
```

Comment:
@zapo this "freezes" part of the auction configs into the adapter, meaning that updates would need a code change to be deployed, to enable parallel (faster) IG auctions. I got the values by looking at what your endpoint returns - but I don't know if they're universal (and I don't know if your endpoint respects the `ae` flag like I do here).

`auctionSignals` is still picked up from the endpoint through `interpretResponse` (as would a few other signals that I don't see populated in your case), but everything in here must be provided synchronously.

Reply:
@dgirardi Looks good, yes they are universal for now. The endpoint does respect `ae` but only when set at top level (initial proposal) `imp.ext.ae`, I need to fix it to also look for `imp.ext.igs.ae`.